dev-python/pipenv/pipenv-2020.4.1_beta_p1.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="Python Development Workflow for Humans"
+HOMEPAGE="https://github.com/pypa/pipenv https://pypi.org/project/pipenv/"
+SRC_URI="https://github.com/pypa/pipenv/archive/v2020.4.1b1.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="test"
+
+PATCHES=(
+	"${FILESDIR}/distable-networked-tests.patch"
+	)
+
+RDEPEND="
+	dev-python/certifi[${PYTHON_USEDEP}]
+	dev-python/virtualenv[${PYTHON_USEDEP}]
+	>=dev-python/virtualenv-clone-0.2.5[${PYTHON_USEDEP}]
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	>=dev-python/pip-18.0[${PYTHON_USEDEP}]
+	>=dev-python/setuptools-36.2.1[${PYTHON_USEDEP}]
+	"
+
+DEPEND="
+	test? (
+		${RDEPEND}
+		<dev-python/pytest-5.0[${PYTHON_USEDEP}]
+		dev-python/pytest-timeout[${PYTHON_USEDEP}]
+		dev-python/pytest-xdist[${PYTHON_USEDEP}]
+		dev-python/flaky[${PYTHON_USEDEP}]
+		dev-python/flask[${PYTHON_USEDEP}]
+		dev-python/mock[${PYTHON_USEDEP}]
+		dev-python/pytz[${PYTHON_USEDEP}]
```

Comment:
Please sort this list by package name, alphabetically.

`mock` and `pytz` and `flask` seem needed indeed but are not communicated in `setup.py`. Has this been brought to upstream's attention?

PS: The fact that they do not support pytest 5 is not cool because it forces Gentoo users to downgrade pytest system-wide and may block installation of other packages. To some degree I am wondering if we even want to enable tests while that problem persists. What do you think?

Reply:
I have made another bug report regarding the tests upstream. But I have not yet added this.
Also, I am quite sure flask is only used for the cli tests which I anyway dropped (the test suite includes a mock of pypi which is built with flask).

Reply:
Still need to sort it alphabetically...

Reply:
Regarding pytest version, I think the majority of pipenv consumers aren't going to run the tests anyway TBH.
Maybe by the time the official release comes they will update the dependency.